.github/workflows/deploy-to-environment.yml

```diff
@@ -62,25 +56,50 @@ jobs:
             echo "CANISTER=${{ github.event.inputs.canister }}" >> $GITHUB_ENV
           fi
 
+      - name: Check release policy
+        run: |
+          if [[ "$NETWORK" ==  "staging" ]] && [[ "${{ github.ref }}" != "refs/heads/main" ]] ; then
```

Comment:
There is no rule that beta may have only head of main.  Indeed, there is an explicit expectation that we should be able to put feature branches and release candidates (that may not be head of main) on beta.

I have proposed that we have a test environment that always holds the latest release candidate (so that would be the latest tag of the form 1.2.3-rc-4), which is effectively what beta has most of the time at the moment, however that has not been approved.

Reply:
ah understood! tks for the clarification!